pkg/config/app/kuma-cp/config.go

```diff
@@ -296,6 +296,7 @@ var DefaultConfig = func() Config {
 		},
 		MeshService: MeshServiceConfig{
 			GenerationInterval: config_types.Duration{Duration: 2 * time.Second},
+			GracePeriod:        config_types.Duration{Duration: 10 * time.Second},
```

Comment:
10s is short. 10s covers simple redeployments.
I was thinking of 1h. It would cover instability if the app completely went down for a couple of mins or more.